sound/soc/sof/core.c

```diff
@@ -301,8 +301,7 @@ int snd_sof_device_probe(struct device *dev, struct snd_sof_pdata *plat_data)
 	if (!sof_ops(sdev) || !sof_ops(sdev)->probe || !sof_ops(sdev)->run ||
```

Comment:
@ranj063   Do we need sof_ops(dev)->stall ? I found sof_ops(sdev)->run here

Reply:
@RanderWang stall is mentioned an optional op right now. So now need to check here.